unity-ggjj/Assets/Scripts/AudioController/AudioController.cs

```diff
@@ -67,22 +66,11 @@ public void PlaySong(AudioClip song, float transitionTime)
     /// Creates a child object that has an AudioSource component
     /// </summary>
     /// <returns>The AudioSource of the new child object</returns>
-    private AudioSource CreateAudioSource(string gameObjectName)
-    {
-        var newGameObject = new GameObject(gameObjectName);
-        newGameObject.transform.parent = transform;
-        return newGameObject.AddComponent<AudioSource>();
-    }
-
-    /// <summary>
-    /// Plays sound effect of desired name.
-    /// </summary>
-    /// <param name="sfx">Name of sound effect asset, must be in `Resources/Audio/SFX`</param>
-    public void PlaySfx(string sfx)
-    {
-        AudioClip soundEffectClip = _narrativeGameState.ObjectStorage.GetObject<AudioClip>(sfx);
-        PlaySfx(soundEffectClip);
-    }
+    private AudioSource CreateAudioSource(string gameObjectName) => new GameObject(gameObjectName) {
+            transform = {
+                parent = transform
+            }
+        }.AddComponent<AudioSource>();
```

Comment:
```suggestion
    private AudioSource CreateAudioSource(string gameObjectName) => new GameObject(gameObjectName) {
            transform = { parent = transform }
        }.AddComponent<AudioSource>();
```
Nit: I can't remember if we had a code style to specify brackets in these situations, and this could definitely be the JS-lover in me, but this seems more readable on one line IMO. What do you think, would that work or is that too not-C#-y?